mm2src/coins/lp_coins.rs

```diff
@@ -3723,6 +3726,8 @@ pub enum MmCoinEnum {
     LightningCoin(LightningCoin),
     #[cfg(feature = "enable-sia")]
     SiaCoin(SiaCoin),
+    #[cfg(feature = "enable-solana")]
+    Solana(solana::SolanaCoin),
```

Comment:
Something I remembered that shortens the time for you in the future is to add Solana here https://github.com/KomodoPlatform/komodo-defi-framework/blob/6085ac4fa847245e4730b97d4cac7937401df567/mm2src/coins/lp_coins.rs#L3865-L3878
Or just move this function to a trait and remove this default implementation. This is of course for next PRs. Hope there is no other things like this that I forgot about, but checking default trait implementations can help you, if not, you will find out while testing for sure.